index.html

```diff
@@ -1,3 +1,22 @@
-<p>test conflikt</p>
-<h1>halo halo</h1>
-<h1>halo halo</h1>
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="styles/main.css">
+</head>
+<body>
+    <header>
+        Header
+    </header>
+    <main>
+        <aside class="main-section">
```

Comment:
Zamiast aside można użyć po prostu tag <section>, który służy do reprezentowania ogólnej samodzielnej sekcji dokumentu, która nie ma bardziej szczegółowego elementu semantycznego. Prawie zawsze powinien w nim znajdować się nagłówek. Jako, że chcemy umieścić w nim tekst, to do samego określenia BLOKU możemy użyć <div>, który dla przykładu otrzyma klasę "main-hero-section". 
<div class="main-hero-section">
  <section class="main-hero-section__content">
    <h1>
    </h1>
    <p>
    </p>
  </section>
  <section class="main-hero-section__image">
    <img />
  </section>
</div>

Oczywiście jest to tylko przykład. Nie powinno się trzymać sztywność jednej reguły, wszystko zależy od przygotowanego przez nas layoutu.